metabase/metabase
# MS1: Open notebook data source in new tab

### Description
This PR addresses the milestone 1 from #46201.
It implements linking to the data source from a notebook mode and opening it in a new tab/window by cmd/ctrl clicking on it (depends on the OS).

### How to verify
1. Open any entity (except Metrics) in the notebook mode (raw table, question, model)
2. Command (meta) or Ctrl click on the data source
3. A new tab/window should open with that source in a simple (chill mode)

Alternatively, run the accompanied E2E tests.

![image](https://github.com/user-attachments/assets/ac632864-23f3-424a-bc98-6f2435a6de08)


### Demo

https://metaboat.slack.com/archives/C0645JP1W81/p1722265651662729?thread_ts=1722257844.614529&cid=C0645JP1W81

### Checklist


### Additional comments
- This PR seems to already implement the milestone 2 (joins) of #46201, but I'll add tests for it separately.
- The tooltip (milestone 3) will also be implemented separately.
- Sandboxing E2E test cases will be implemented in a follow-up PR.
